spec/features/admin/chapters_spec.rb

```diff
@@ -20,4 +20,63 @@
       expect(page).to have_content("Chapter codebar Brighton has been successfully created")
     end
   end
+
+  context "#editing a chapter" do
+    let(:chapter) { Fabricate(:chapter) }
+
+
```

Comment:
@notthepoint It would be nice to do away with the extra empty lines here and similarly in other places. What do you think ?